src/renderer/components/MultiPathInput.tsx

```diff
@@ -27,18 +28,18 @@ export const MultiPathInput: React.FC<MultiPathInputProps> = ({ paths, updatePat
       return false;
     }
 
-    if (newPath.includes(rootFolder)) {
+    if (isSubdir(rootFolder, newPath)) {
       addErrorToast("Cannot add sub directories of the Root SLP Directory.");
       return false;
     }
 
     let pathsToCheck = paths;
     for (let i = 0; i < pathsToCheck.length; i++) {
       const path = pathsToCheck[i];
-      if (newPath.includes(path)) {
+      if (isSubdir(path, newPath)) {
```

Comment:
Opted to also call `isSubdir` in other spots where the subdirectory check was being done. The tests should validate that the functionality still works as it originally did.